Doc/library/socket.rst

```diff
@@ -631,6 +647,18 @@ Constants
 
    .. versionadded:: 3.12
 
+.. _socket-ethernet-types:
+
+.. data:: ETHERTYPE_ARP
+          ETHERTYPE_IP
+          ETHERTYPE_IPV6
+          ETHERTYPE_VLAN
+
+   IEEE 802.3 protocol numbers [1]_.
+
```

Comment:
Assuming Linux, FreeBSD and macOS doesn't just count as "Unix", we still need to document this:

```suggestion

.. availability:: Linux, FreeBSD, macOS.

```